docs/releases/v2_0_0.md

```diff
@@ -0,0 +1,153 @@
+# Release 2.0.0
+
+## Summary
+This is Tern's second major release. Our CLI has changed since the last release to account for a new feature and two new CLI options:
+* The ability to "lock" a provided Dockerfile (`tern lock <path_to_Dockerfile> will produce `Dockerfile.lock`). See "New Features" section below for more details.
```

Comment:
`tern lock <path_to_Dockerfile>` is missing a backtick

Reply:
Nice catch.